package.json

```diff
@@ -5,6 +5,7 @@
         "bootstrap": "lerna bootstrap",
         "lint:staged": "lerna run lint:staged --stream",
         "lint": "lerna run lint --stream",
+        "manual-version": "lerna version --preid=rc --no-push --no-changelog --no-private",
         "version": "lerna bootstrap --ignore-scripts -- --package-lock-only --no-audit && git add packages/*/package-lock.json",
```

Comment:
how about replacing with the original "version" script to avoid confusion?

Reply:
this 'version' is removed, already sync up wth @LongOddCode